docs/getting-started/user-guides/testnet-voting.md

```diff
@@ -0,0 +1,183 @@
+# Testnet Hard Fork Voting Guide
+
+This guide was last updated for v0.8.2 
+
+As of the v0.8.0 release of Feb. 13, 2017, the hard-fork voting mechanism was implemented for testing on the Decred Testnet. The test vote is estimated to start on Feb 25, 2017 around 1PM CST and last for 7 days. If you wish to participate, guides for Paymetheus and the Command-Line applications can be found here. 
+
+---
+
+## Introduction
+
+There is a two-step process for voting to implement consensus changes that would create a hard fork. Note: the following block intervals are for the testnet, they will differ for mainnet.
+
+The first step is to meet the upgrade threshold on the network. After the hard fork code is released, all of the nodes on the network participating in PoW/PoS need to first upgrade before the voting can begin. For Proof-of-Work, at least 75% of the 100 most recent blocks must have the latest block version. For Proof-of-Stake, 75% of the votes cast within a static 2016 block interval must have the latest votebit version.
```

Comment:
After the hard fork code is released, all of the nodes on the network participating in PoW/PoS need to first upgrade before the voting can begin. -->  s/all of/majority of/